docs-src/tutorials/02-usage.md

```diff
@@ -157,15 +157,15 @@ created.
 - `title`: The step's title. It becomes an `h3` at the top of the step.
 - `attachTo`: What element the step should be attached to on the page.
 It should be an object with the properties `element` and `on`, where `element` is an element selector string
-or a DOM element and `on` is the optional direction to place the Tippy tooltip.
-              
+or a DOM element and `on` is the optional direction to place the step element.
```

Comment:
```suggestion
or a DOM element and `on` is the optional direction to place the Popper tooltip.
```

Reply:
Popper is not really a "tooltip", but can do 👍

Reply:
We're creating tooltips with Popper though

Reply:
True :)